diff --git a/dist/esm/plugin.js b/dist/esm/plugin.js
index 0d6b81aec5dbaefc8297a21d4721204ca89cb7e3..1ae42c0729f56c5609c10cce7964ee5bf24538a9 100644
--- a/dist/esm/plugin.js
+++ b/dist/esm/plugin.js
@@ -150,11 +150,11 @@ function TanStackStartVitePluginCore(corePluginOpts, startPluginOpts) {
                   ...result.optimizeDeps.exclude.sort(),
                   ...additionalOptimizeDeps.exclude,
                   `@tanstack/${corePluginOpts.framework}-start/server`
-                ],
+                ].filter(dep => !dep.startsWith("@apollo/")),
                 include: [
                   ...additionalOptimizeDeps.include,
                   ...result.optimizeDeps.include.sort()
-                ]
+                ].filter(dep => !dep.startsWith("@apollo/")),
               }
             }
           },
@@ -177,8 +177,8 @@ function TanStackStartVitePluginCore(corePluginOpts, startPluginOpts) {
               startPackageName,
               ...result.optimizeDeps.exclude.sort(),
               ...additionalOptimizeDeps.exclude
-            ],
-            include: [...additionalOptimizeDeps.include]
+            ].filter(dep => !dep.startsWith("@apollo/")),
+            include: [...additionalOptimizeDeps.include].filter(dep => !dep.startsWith("@apollo/"))
           },
           /* prettier-ignore */
           define: {
